sdk/appconfiguration/Azure.Data.AppConfiguration/src/AuthenticationPolicy.cs

```diff
@@ -23,62 +23,67 @@ public AuthenticationPolicy(string credential, byte[] secret)
             _secret = secret;
         }
 
-        public override async ValueTask ProcessAsync(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
+        public override void Process(HttpMessage message, ReadOnlyMemory<HttpPipelinePolicy> pipeline)
```

Comment:
Since the TODO has been removed and we're so close to P5, let's not refactor this now to reduce risk.  I like the refactoring, though - @AlexanderSher, could you file an issue to do this refactoring in P6?

Reply:
Done: https://github.com/Azure/azure-sdk-for-net/issues/8294